tox.ini

```diff
@@ -71,4 +71,3 @@ commands =
     pytest -rA tests/test_web/test_tidy3d_material_library.py    
     pytest -rA tests/test_web/test_tidy3d_task.py
     pytest -rA tests/test_web/test_webapi.py
```

Comment:
Can't we just `pytest -ra tests` here? Or are we excluding a few tests?

Reply:
initially we excluded some tests, but after I reorganized the tests, we put the ones we want to exclude in `_test_local` which gets ignored by pytest so yea I think we can safely do this. changed